frontend/src/views/SQR/Form.vue

```diff
@@ -296,7 +296,7 @@ export default class Form extends Vue {
     [FormState.UNSAVED_CHANGES]: "red",
     [FormState.ERROR]: "red"
   };
-  draftStatus = FormState.INITIAL_LOAD;
+  formStatus = FormState.INITIAL_LOAD;
```

Comment:
Inconsistency is there: `Status` vs `State`.
```suggestion
  formState = FormState.INITIAL_LOAD;
```